### PR TITLE
update diagrams based on teacher feedback

### PR DESCRIPTION
**Pull request by kennethszj**
1. change class diagram to use + and -
2. add colon for entities in sequence diagram

### CODE REVIEW

File: docs/images/TagPatientClassDiagram.png
Comment (NCF3535) [inline docs/images/TagPatientClassDiagram.png:1]: Should the new class diagram still have the square and circles as compared to the lecture notes? Could try using `!include style.puml`

Comment (NCF3535) [inline docs/images/TagPatientClassDiagram.png:1]: Shouldn't the color of the classes follow the rest in being colored according to the architecture diagrams?

File: docs/diagrams/CompletionRateSequenceDiagram.puml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 @startuml
-actor User as user
-participant Ui
-participant Hospital
-participant Patient
-participant TaskList
+actor ":User" as user
```
Comment (NCF3535) [inline docs/diagrams/CompletionRateSequenceDiagram.puml]: Should add the color codes for the sequence diagrams based on the ones in style.puml for standardisation

File: docs/images/CompletionRateSequenceDiagram.png
Comment (NCF3535) [inline docs/images/CompletionRateSequenceDiagram.png:1]: Shouldn't the color of the classes follow be colored according to the architecture diagrams?

**Review by NCF3535 — CHANGES_REQUESTED** [review]
Overall the edits do address concerns in #112 but did not follow what was agreed on.

File: docs/images/TagPatientClassDiagram.png
Comment (kennethszj) [reply docs/images/TagPatientClassDiagram.png:1]: have change it to use style puml

File: docs/images/StateSwitchingSequenceDiagram.png
Comment (NCF3535) [inline docs/images/StateSwitchingSequenceDiagram.png:1]: Shouldn't return statements be dotted lines as per #112?

File: docs/images/TagPatientClassDiagram.png
Comment (NCF3535) [inline docs/images/TagPatientClassDiagram.png:1]: Should there be an arrow between `Command` and `AddPatientCommand`?

**Review by NCF3535 — COMMENTED** [review]
Just minor fixes more and all should be good!

**Review by NCF3535 — APPROVED** [review]
LGTM, good job on the changes

**Pull request merged**